Sources/FluentMongoDriver/MongoDB+Join.swift

```diff
@@ -95,8 +95,23 @@ extension DatabaseQuery {
                         "$unwind": "$\(alias ?? schema)"
                     ]))
                 case .custom:
-                    fatalError()
+                    throw FluentMongoError.unsupportedJoin
```

Comment:
Huge plus

Reply:
Agreed - too many fatal errors all over Fluent as it is